.github/workflows/qa.yml

```diff
@@ -0,0 +1,12 @@
+on: [push, pull_request]
+name: Quality assurance
+jobs:
+    phpstan:
+        name: PHPStan
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@master
+            - name: PHPStan
+              uses: "docker://oskarstark/phpstan-ga"
```

Comment:
@0.12.2 ?

Reply:
Is there a reason, why we shouldn't use the latest version? We're doing the same for every other QA tool (php-cs-fixer, phpunit, ...)

Reply:
Ok in this case

Reply:
What about the PHP extension mongodb? can I do something?